src/Groups/matrices/MatGrp.jl

```diff
@@ -435,6 +435,7 @@ function Base.:^(x::MatrixGroupElem, y::MatrixGroupElem)
 end
 
 comm(x::MatrixGroupElem, y::MatrixGroupElem) = inv(x)*conj(x,y)
+#T why needed? GAPGroupElem has x^-1*x^y
```

Comment:
If we don't need it, remove it? (Though perhaps in a separate PR?)

Though then again, now I wonder if the general one shouldn't be re-implemented by calling `GAP.Globals.Comm`? And then it might make sense to keep the above method here... Or even change it to, say, `inv(y*x)*(x*y)` (one less inversion, which is arguably the most expensive bit here?)

Reply:
Let us leave this to another pull request.